### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by ericanaughton**
:evergreen_tree: 

Ready to go.

This PR includes the changes suggested by @m5rk on the previous PR https://github.com/LMHT/LMHT.github.io/pull/100.  Also, some rewording in the README and new found typos.

### CODE REVIEW

**Comment by whazzmaster** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # LMHT - HTML: Lenguaje de Marca de Hipertexto - Hyper Text Markup Language
 
-Welcome to [LMHT].  This is an open source code repository to help Spanish speakers translate and understand HTML tags.  Originally started as a project by Team Open Brace in the [YWebCA] Summer of 2015 class, this project is now being developed further as an open source project by [Adorable] to expand it's possibilities.  With your help, we can expand this into an excellent learning tool for bilingual programmers.  
+Welcome to [LMHT].  This is an open source code repository to help Spanish speakers translate and understand HTML tags.  LMHT was originally started as a [YWebCA] project by Team Open Brace in the Summer of 2015. [Adorable] and other contributors are working towards expanding this project into an valuable learning tool for bilingual programmers.
```
Comment (m5rk) [inline README.md]: Typo: `into a [rather than an] valuable learning tool`

**Comment by m5rk** [comment]
:+1: - Just a typo.

**Comment by ericanaughton** [comment]
thanks @m5rk and @whazzmaster !

**Pull request merged**